x-pack/plugins/triggers_actions_ui/public/application/sections/alert_form/alert_form.tsx

```diff
@@ -81,11 +81,15 @@ export const AlertForm = ({ alert, canChangeTrigger = true, dispatch, errors }:
   );
 
   const [alertTypesIndex, setAlertTypesIndex] = useState<AlertTypeIndex | undefined>(undefined);
-  const [alertInterval, setAlertInterval] = useState<number>(
-    alert.schedule.interval ? parseInt(alert.schedule.interval.replace(/^[A-Za-z]+$/, ''), 0) : 1
+  const [alertInterval, setAlertInterval] = useState<number | undefined>(
+    alert.schedule.interval
+      ? parseInt(alert.schedule.interval.replace(/^[A-Za-z]+$/, ''), 0)
```

Comment:
It seems like this will only change a string that has alpha chars in it, to '', because of the leading/trailing `^$` in the pattern.

Rather than do this replace, and the one below, I wonder if we should enhance  https://github.com/elastic/kibana/blob/master/x-pack/plugins/alerting/common/parse_duration.ts to do some of this work - I moved that to common a while back for some other UI need.  It would be nice to keep all the duration parsing stuff in one place.  One thought would be a new function that just split the number and units, returned `{ value: number; units: string }` or something.